langs/elm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.16 as builder
+
+RUN wget -O - 'https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz' \
+    | gunzip -c >/usr/local/bin/elm
+RUN chmod +x /usr/local/bin/elm
+RUN apk add --no-cache nodejs=16.17.1-r0
```

Comment:
Why not use the latest LTS `18.12.0`?

Reply:
`16.17.1-r0` is the highest available version on alpine 3.16

https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.16